docs/reference.rst

```diff
@@ -1703,6 +1704,58 @@ If a value is passed for the :class:`RelatedFactory` attribute, this disables
                       main_lang=factory.SelfAttribute('..lang'),
                   )
 
+RelatedFactoryList
+""""""""""""""""""
+
+.. class:: RelatedFactoryList(factory, factory_related_name='', length=2, **kwargs)
+
+    .. OHAI_VIM**
+
+    A :class:`RelatedFactoryList` behaves like a :class:`RelatedFactory`, only it returns a
+    list of factories. This is useful for simulating one-to-many relations, rather than the
+    one-to-one relation generated by :class:`RelatedFactory`.
+
+
+    .. attribute:: factory
+
+        As for :class:`SubFactory`, the :attr:`factory` argument can be:
+
+        - A :class:`Factory` subclass
+        - Or the fully qualified path to a :class:`Factory` subclass
+          (see :ref:`subfactory-circular` for details)
+
+    .. attribute:: name
+
+        The generated object (where the :class:`RelatedFactory` attribute will
+        set) may be passed to the related factories if the :attr:`factory_related_name` parameter
+        is set.
+
+        It will be passed as a keyword argument, using the :attr:`name` value as
+        keyword:
+
+    .. attribute:: length
```

Comment:
I suggest using already existing terms, for `create_batch`, `build_batch` etc it is `size`